api/src/utils.py

```diff
@@ -0,0 +1,46 @@
+import os
```

Comment:
This entire file can and should be shared across `model`, `app` and `api`.

Reply:
making 2 docker images to share a common parent utils folder is a bit hard to setup.
as an easier workaround, I simply created a `common.py` to contain common util function, and then duplicate them into `app/src` and `api/src`